package.json

```diff
@@ -22,7 +22,11 @@
   },
   "homepage": "https://github.com/FictionalDevelopers/expenses-tracker#readme",
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.25",
+    "@fortawesome/free-solid-svg-icons": "^5.11.2",
+    "@fortawesome/react-fontawesome": "^0.1.5",
```

Comment:
minor: generally it is more preferred to use [@material-ui/icons](https://material-ui.com/getting-started/installation/#svg-icons) package for icons in material ui ecosystem